website/case-studies/implementation-of-flint-modules.md

```diff
@@ -0,0 +1,407 @@
+id:  implementation-of-flint-modules
+title:  Implementation of FLINT Modules
+slug:  /implementation-flint-modules
+description:  Know and understand the FLINT Modules and how they work, along with the contribution of each FLINT Module
+image: https://i.imgur.com/An53MDO.png
+keywords:
+- FLINT Modules
+- FLINT
+- Modules
+- GCBM
+- Hybrid Forest
+- Empirical Growth
+- Turnover module
+- Decomposition module
+- Chapman Richards
+- Carbon cycle
+- moja global
+- WOFOST crop growth
+---
+
+![Cover Image for the case study titled `Implementation of FLINT Modules`](../static/img/implementation-of-flint-modules.png)
+
+FLINT is a modular tool that supports better land management and helps reduce greenhouse gas (GHG) emissions worldwide, and allows multiple models and data required to better understand land management to be brought together in one place. 
+
+A module is a self-contained set of operations that determine the state of, or change in, variables across a specified period of time for a single Simulation Unit in direct response to event notifications from the FLINT core system (Unit Controller). For example, the empirical forest growth module includes all the operations required to simulate biomass accumulation.
+
+Modules are all written to work with the [`develop`](https://github.com/moja-global/FLINT/tree/develop) branch of the **FLINT** and any actively maintained branches will work with the most recent version (and are broken with old enough builds of the FLINT). The FLINT Application Programming Interface (API) has evolved over time, so some modules might be out of date. None of these modules require a custom fork of FLINT.
+
+## Background
+
+It is useful to think about what a module manifest might look like. **Manifests** are meta-data used by the package managers to describe the compatibility (FLINT versions that are used), dependencies (necessary inputs) and the configuration requirements (for example whether the module uses daily, monthly or annual time steps).
+
+This case study describes the core contribution of each module.
+
+## Index
+
+1.  [Prerequisites](#prerequisites)
+2.  [How do modules work?](#how-do-modules-work)
+3.  [Empirical forest growth module](#empirical-forest-growth-module)
+4.  [Hybrid forest growth module](#hybrid-forest-growth-module)
+5.  [WOFOST crop growth module](#wofost-crop-growth-module)
+6.  [Turnover module](#turnover-module)
+7.  [Decomposition module](#decomposition-module)
+8.  [RothC soil carbon module](#rothc-soil-carbon-module)
+9.  [Chapman Richards module](#chapman-richards-module)
+10. [GCBM module](#gcbm-module)
+11. [Frequently asked questions](#frequently-asked-questions)
+
+## Prerequisites
+
+These are the prerequistes you need to know before learning about **Modules**. This section presumes that you are already familiar with **FLINT**. If not, see [the FLINT repository](https://github.com/moja-global/FLINT) to better understand **FLINT** before continuing.
+
+## How do modules work?
```

Comment:
Move this and Prerequisites into 'Background' so that your index only describes the different modules